runtime/interpreter/interpreter.go

```diff
@@ -2226,13 +2403,15 @@ type ValueConverterDeclaration struct {
 var ConverterDeclarations = []ValueConverterDeclaration{
 	{
 		name:         sema.IntTypeName,
+		targetType:   sema.IntType.Tag(),
```

Comment:
Why do only `Int` and `UInt` need the `targetType`, why is it not needed for the other types?

Reply:
whoops dead code